gamedata/configs/text/eng/srp_strings.xml

```diff
@@ -31,13 +31,13 @@
 		<text>You must wait for the transmission to finish.</text>
 	</string>
 	<string id="tips_wait_emission_finish">
-		<text>You must wait for the emission to finish.</text>
+		<text>It's not safe to travel during the emission.</text>
```

Comment:
`'` should be replaced by `’` to be consistent with the style followed in the SRP. With that said, I don't actually remember why I globally replaced the former with the latter in the English texts. It may have been an attempt to fix [this bug](https://steamcommunity.com/app/20510/discussions/0/1638676181986370677/).